migration/matchPopulation/reprocessMatches.groovy

```diff
@@ -0,0 +1,54 @@
+@Grab('io.github.http-builder-ng:http-builder-ng-core:1.0.4')
+import groovy.json.JsonSlurper
+import java.util.logging.Logger
+
+import static groovyx.net.http.HttpBuilder.configure
+import static groovyx.net.http.util.SslUtils.ignoreSslIssues
+
+/**
+ * Simple script to reprocess matches for the given list of reference/purpose ids
+ *
+ * Requires an authentication as an admin user.
+ *
+ * Sample Usage:
+ *   groovy reprocessMatches.groovy \
+ *     `gcloud auth print-access-token` \
+ *     https://local.broadinstitute.org:27443
+ */
+
+reprocessMatches(args[0], args[1])
+
+static void reprocessMatches(String authToken, String uriHost) {
+    Logger logger = Logger.getLogger("ReprocessMatches")
+
+    //list of purposeIds to reprocess matches for, comma separated list exported from postico
+    def purposeIds = ['fb885dcf-8639-44a7-97c4-0383a4ebd75c', '0caad95b-37c1-47fd-9e47-ac2b96fd70dd', 'd9a17040-cd35-4983-a9a1-fc080a691bb0']
```

Comment:
Can we pull these in as a file, one id per line? [File handling in groovy is pretty simple](https://stackoverflow.com/questions/7729302/how-to-read-a-file-in-groovy-into-a-string).